kratos/python_scripts/linear_solver_factory.py

```diff
@@ -305,11 +305,6 @@ def ConstructSolver(configuration):
         scaling = params["scaling"].GetBool()
 
         linear_solver = AMGCL_NS_Solver(params)
-    #
```

Comment:
can't we emulate the interface so that the corresponding interface through Eigen is used?

Reply:
sure, just a question whether I should or not.
But if you ask for it I can do it

Reply:
I mean, what is your suggestion?

Reply:
Dont have a better suggestion, I followed your proposal

Btw if you want to use the pardiso why not using it through Eigen?